main.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-tools/go-steputils/stepconf"
+)
+
+type config struct {
+	Platform string `env:"platform,opt[ios,android,'ios,android']"`
+	Readd    bool   `env:"readd_platform,opt[true,false]"`
+	Version  string `env:"cordova_version"`
+	WorkDir  string `env:"workdir,dir"`
+}
+
+func runCommand(name string, arg ...string) error {
+	log.Infof("$ %s %s", name, strings.Join(arg, " "))
+	cmd := exec.Command(name, arg...)
```

Comment:
we have `command.NewWithXyz` call so you can skip setting STDs, also you can just simply return with that.
And if you function is one liner, do you need the function separated?